src/CMakeLists.txt

```diff
@@ -105,6 +105,7 @@ set(mc_rtc_utils_HDR
   ../include/mc_rtc/DataStore.h
   ../include/mc_rtc/type_name.h
   ../include/mc_rtc/debug.h
+  ../include/mc_rtc/pragma.h
```

Comment:
The file should also be added in `debian/libmc-rtc-dev.install`

Reply:
Done in 0e665f5